sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample8_MockClient.cs

```diff
@@ -15,15 +16,19 @@ public void MockClient()
         {
             // Create a mock response
             var mockResponse = new Mock<Response>();
+
             // Create a client mock
             var mock = new Mock<ConfigurationClient>();
+
             // Setup client method
-            mock.Setup(c => c.Get("Key", It.IsAny<string>(), It.IsAny<DateTimeOffset>(), It.IsAny<CancellationToken>()))
+            // TODO: why does this not work?
```

Comment:
If the "it doesn't work" issue is that it doesn't match I've had issues in the past with Moq not being okay with intermixing literal arguments with matcher arguments (like It).

So perhaps you just want to to do c.Get(It.Is<string>(key == "Key"))?